src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/Roslyn.SyntaxVisualizer.Control.csproj

```diff
@@ -7,6 +7,8 @@
     <IsProductComponent>false</IsProductComponent>
     <UseWpf>true</UseWpf>
     <UseWindowsForms>true</UseWindowsForms>
+    <Nullable>enable</Nullable>
+    <LangVersion>9</LangVersion>
```

Comment:
💡 This property should be defined further up, where it can apply to all projects.